### DISCUSSION TITLE
allowing certain domans for all clients

### DISCUSSION THREAD

**Discussion by allhailCdosdude** [AdguardTeam/AdGuardHome#5349] [Q&A]
Because of some rules, certain domains, for example below, are blocked. 

[https://www.delta.com/eu/en](url)
[https://www.momondo.de/?ispredir=true](url)
[https://onlinebanking.affinityplus.org/SignIn.aspx](url)

I want all clients to be able to access these websites. 
I'm using version Version: v0.107.6.

Can someone please give example(s) of how I can do this, please, under "Custom Filtering Rules?"
Thanks a lot in advance.

**Comment by ammnt** [comment]
> [https://www.delta.com/eu/en](url)
> [https://www.momondo.de/?ispredir=true](url)
> [https://onlinebanking.affinityplus.org/SignIn.aspx](url)

Try these rules:
@@||delta.com^$important
@@||momondo.de^$important
@@||onlinebanking.affinityplus.org^$important

**Reply by allhailCdosdude** [reply]
thanks for that. but they did not work, even after restarting the router. I used them exactly as you suggested. 
I noticed that [www.researchgate.net.cdn.cloudflare.net](url) was blocked. But I unblocked it from the Query Log page. And it now works. When I looked at the Custom Filtering Rule page "www" was part of the new rule allowing researchgate.
And when I tested your rules using the 'Check' function/button on the Custom Filtering Rule page, it reports:
"Not found in your filter lists"
But when I do the same with researchgate, it reports:
"www.researchgate.net.cdn.cloudflare.net
@@||www.researchgate.net.cdn.cloudflare.net^$important
Custom filtering rules"
And below that a red button with "Block"
So, the only change I notice is that researchgate is "www" and the other sites are "https"
Could this be the issue?

**Reply by ammnt** [reply]
> thanks for that. but they did not work, even after restarting the router. I used them exactly as you suggested. any other ideas?

Yup. Clear the cache on all devices, then make a few queries for these hosts and show the logs here.
